docs/client_reference.rst

```diff
@@ -515,6 +528,9 @@ The client session supports the context manager protocol for self closing.
 
          .. versionadded:: 2.3
 
+      :param bool remote_socks_resolve: Resolve destination host on the remote socks server.
```

Comment:
Sorry, I still not follow what is the difference between different `remote_socks_resolve` values.
`True` for host DNS resolution on socks server. `False` for DNS resolution on client?
When user should provide non-default parameter?

Reply:
> Fingerprint is checked for transport's certificate.

Look at this test:
https://github.com/nibrag/aiohttp/blob/90bf6d9c20570e3e05c92ab02a69ee3c0d8818d7/tests/test_proxy_functional.py#L672-L692

Reply:
> True for host DNS resolution on socks server. False for DNS resolution on client?
Yes. Docs updated.